app/src/main/res/drawable/bottom_sheet_rounded_background.xml

```diff
@@ -1,8 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
What are these changes for?

Reply:
Border/shadow is only top the shape. Instead of coloring.

Reply:
Maybe of course a more effective method to achieve this. 

And external backgroundTint replaces the border  from this method.

Reply:
Just leave this without a border, it's fine that it blends in the library filter sheet